app/controllers/crime_applications_controller.rb

```diff
@@ -29,4 +29,24 @@ def destroy
   end
 
   def confirm_destroy; end
+
+  private
+
+  def order_param
+    return 'created_at' unless ordering_param_allowed?
+
+    params[:order]
+  end
+
+  def ordering_param_allowed?
+    %w[created_at].include? params[:order]
+  end
+
+  def sort_param
```

Comment:
Wondering if some of these methods are common to both controllers, the in progress applications (`crime_applications`) and the submitted/returned one (`completed_crime_applications`) in which case ideally we would want to raise those methods to the super class of both controllers, `dashboard_controller` to reuse as a common interface.